scripts/install/installer

```diff
@@ -1593,13 +1582,46 @@ def _update_cluster(ctx, development):
     cluster.update()
 
 
+@cluster.command("precheck")
+@click.pass_context
+def preFlightCheck(ctx):
+    """
+    Check if the environment is ready for upgrade
+    """
+
+    config = ctx.obj["config"]
+    enabled_pnodes = get_enabled_pnodes(config)
+    pnodes = get_pnodes(
+        config, versions=ctx.obj["versions"], roles=["cpu", "storage", "controller"]
+    )
+
+    offline_nodes = list()
+    for node in pnodes:
+        if node.name in enabled_pnodes:
+            if not node.is_up():
+                offline_nodes.append(node.name)
+
+    if offline_nodes:
+        print(
+            "Can't connect to the following nodes: {}. Please check or put them info maintenance state".format(
```

Comment:
Better message would be: "Either investigate the issue or put the node(s) in maintenance".